Lesson5

vmayv opened the pull request.

vmayv committed:
Добавлен FluentMigrator
Добавлена папка Migrations

vmayv committed:
Задействован FluentMigrator

vmayv committed:
Добавлена библиотека Quartz

vmayv committed:
-

vmayv committed:
Добавлены классы задач

vmayv committed:
Добавлены System.Diagnostics

vmayv committed:
-

vmayv committed:
Репозитории добавлены как AddSingletone вместо AddScoped

vmayv committed:
Добавлен код работы задач во все метрики, кроме озу

vmayv committed:
Добавлен perfornamce counter для ram
job также заполнен

Forkosigan commented on MetricsAgent/DAL/Migrations/DBMigration.cs:
```diff
@@ -0,0 +1,49 @@
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MetricsAgent.DAL.Migrations
```
напишу тут общие замечания по коду. Не по этому клаксс а вообще
1)  Я вижу есть проект с названием Core. Но при этом вон папке ClassLibrary. Это непорядок с точки зрения структуры кода. Проект core  - в папке Core
2) Зачем enum percentile внутри класса c название Class? Сам по себе enum это уже класс (просто специальный вид класса) и его не надо вкюбчать в класс. А файл переименовать в Percentile.cs
3) Вижу кучу кусков закомментированного кода. Это smell. Так делать не надо. Все лишнее - удаляется. 
А так в целом все ок

vmayv replied:
Поправил все замечания в d94cc405d322dd71c02795cc0d4cf5fb3aa1ff6e

vmayv committed:
Папка ClassLibrary переименована в Core
Поправлены ошибки в using
Поправлены все ошибки использования Percentile из-за замены ClassLibrary на Core
